src/Uno.UI/UI/Xaml/Media/Animation/KeyTime.cs

```diff
@@ -66,5 +66,12 @@ public override string ToString()
 		{
 			return this.TimeSpan.ToXamlString(CultureInfo.InvariantCulture);
 		}
+
+		int IComparable<KeyTime>.CompareTo(KeyTime other) => TimeSpan.CompareTo(other.TimeSpan);
+
+		public static bool operator <(KeyTime keytime1, KeyTime keytime2) => keytime1.TimeSpan < keytime2.TimeSpan;
+		public static bool operator >(KeyTime keytime1, KeyTime keytime2) => keytime1.TimeSpan > keytime2.TimeSpan;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement alternative method 'Compare' for the operator '>'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)